MANIFEST.in

```diff
@@ -0,0 +1,9 @@
+global-include *.rst
+include LICENSE
+include COPYING
+graft docs
+graft tests/unit
+include examples/haas.cfg
+include scripts/haas_network.service
```

Comment:
Given that your systemd pr isn't merged yet, this probably needs to wait on that.